### DISCUSSION TITLE
Having this error on Docker install of Plausible [error] Ch.Connection (#PID<0.3371.0>) failed to connect: ** (Mint.TransportError) non-existing domain

### DISCUSSION THREAD

**Discussion by bastianhilton** [plausible/analytics#3633] [Self-Hosted Support]
Having this error when attempting to install plausible via Docker. 

`23:57:31.129 [error] Ch.Connection (#PID<0.3373.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.129 [error] Ch.Connection (#PID<0.3378.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.129 [error] Ch.Connection (#PID<0.3375.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.129 [error] Ch.Connection (#PID<0.3377.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3396.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3392.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3388.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3387.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3386.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3385.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3384.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3380.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3379.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3376.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3374.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3372.0>) failed to connect: ** (Mint.TransportError) non-existing domain
23:57:31.130 [error] Ch.Connection (#PID<0.3371.0>) failed to connect: ** (Mint.TransportError) non-existing domain
{"Kernel pid terminated",application_controller,"{application_start_failure,plausible,{{shutdown,{failed_to_start_child,'Elixir.Plausible.Session.Salts',{#{'__exception__' => true,'__struct__' => 'Elixir.Postgrex.Error',connection_id => 3117,message => nil,postgres => #{code => undefined_table,file => <<\"parse_relation.c\">>,line => <<\"1384\">>,message => <<\"relation \\"salts\\" does not exist\">>,pg_code => <<\"42P01\">>,position => <<\"13\">>,routine => <<\"parserOpenTable\">>,severity => <<\"ERROR\">>,unknown => <<\"ERROR\">>},query => <<\"DELETE FROM \\"salts\\" AS s0 WHERE (s0.\\"inserted_at\\" < now() - '48 hours'::interval)\">>},[{'Elixir.Ecto.Adapters.SQL',raise_sql_call_error,1,[{file,\"lib/ecto/adapters/sql.ex\"},{line,913},{error_info,#{module => 'Elixir.Exception'}}]},{'Elixir.Ecto.Adapters.SQL',execute,6,[{file,\"lib/ecto/adapters/sql.ex\"},{line,828}]},{'Elixir.Plausible.Session.Salts','-start_link/1-fun-0-',0,[{file,\"lib/plausible/session/salts.ex\"},{line,8}]},{'Elixir.Agent.Server',init,1,[{file,\"lib/agent/server.ex\"},{line,8}]},{gen_server,init_it,2,[{file,\"gen_server.erl\"},{line,851}]},{gen_server,init_it,6,[{file,\"gen_server.erl\"},{line,814}]},{proc_lib,init_p_do_apply,3,[{file,\"proc_lib.erl\"},{line,240}]}]}}},{'Elixir.Plausible.Application',start,[normal,[]]}}}"}
Kernel pid terminated (application_controller) ({application_start_failure,plausible,{{shutdown,{failed_to_start_child,'Elixir.Plausible.Session.Salts',{#{'__exception__' => true,'__struct__' => 'Elixir.Postgrex.Error',connection_id => 3117,message => nil,postgres => #{code => undefined_table,file => <<"parse_relation.c">>,line => <<"1384">>,message => <<"relation \"salts\" does not exist">>,pg_code => <<"42P01">>,position => <<"13">>,routine => <<"parserOpenTable">>,severity => <<"ERROR">>,unknown => <<"ERROR">>},query => <<"DELETE FROM \"salts\" AS s0 WHERE (s0.\"inserted_at\" < now() - '48 hours'::interval)">>},[{'Elixir.Ecto.Adapters.SQL',raise_sql_call_error,1,[{file,"lib/ecto/adapters/sql.ex"},{line,913},{error_info,#{module => 'Elixir.Exception'}}]},{'Elixir.Ecto.Adapters.SQL',execute,6,[{file,"lib/ecto/adapters/sql.ex"},{line,828}]},{'Elixir.Plausible.Session.Salts','-start_link/1-fun-0-',0,[{file,"lib/plausible/session/salts.ex"},{line,8}]},{'Elixir.Agent.Server',init,1,[{file,"lib/agent/server.ex"}
Crash dump is being written to: erl_crash.dump...done`

I'm using Plausible latest image from hub.docker.com
using Portainer 2.19.3
using Mariadb database hosted on AWS

Any ideas what is happening?

**Comment by ruslandoga** [comment]
👋 @bastianhilton 

The ClickHouse error means that Plausible can't resolve the domain provided for it and the PostgreSQL error means the `salts` table doesn't exit. Both might be the result of bad configuration. Would you be able to post the configuration files you are using with Portainer?

> Mariadb

Plausible doesn't support any databases other than ClickHouse (used for analytics) and PostgreSQL (used for everything else).

**Comment by bastianhilton** [comment]
ok thank you I didn't have a clickhouse database setup and Plausible is on Postgres, posted Mariadb by mistake.
